testflinger/api/v1.py

```diff
@@ -26,6 +26,11 @@
 import pkg_resources
 from flask import current_app, jsonify, request, send_file
 from werkzeug.exceptions import BadRequest
+from prometheus_client import Counter
+
+
+jobs_metric = Counter("jobs", "Number of jobs", ["queue"])
+reservations_metric = Counter("reservations", "Number of reservations")
```

Comment:
I'm not sure about that, but should there be something like `registry.register` after the metrics definition?

Reply:
I think this is only needed if you are registering a custom collector